pkg/operation/shoot/shoot_test.go

```diff
@@ -560,4 +562,52 @@ var _ = Describe("shoot", func() {
 			)))
 		})
 	})
+	Describe("HA tests", func() {
+		var shoot *Shoot
+		BeforeEach(func() {
+			shoot = &Shoot{}
+			shoot.SetInfo(&gardencorev1beta1.Shoot{})
+		})
+		Context("IsHAControlPlaneConfigured using annotation", func() {
+			It("HA annotation is set", func() {
+				shoot.GetInfo().ObjectMeta.Annotations = map[string]string{
+					v1beta1constants.ShootAlphaControlPlaneHighAvailability: v1beta1constants.ShootAlphaControlPlaneHighAvailabilityMultiZone,
+				}
+				Expect(shoot.IsHAControlPlaneConfigured()).To(BeTrue())
+			})
+			It("HA annotation is not set", func() {
+				Expect(shoot.IsHAControlPlaneConfigured()).To(BeFalse())
+			})
+		})
+		Context("IsHAControlPlaneConfigured using Shoot ControlPlane spec", func() {
```

Comment:
This context should be moved into the

```
Describe("#IsHAControlPlaneConfigured", func() {
```

block since it's still testing the same function.